### PR TITLE
Issue 303 fixes

### PR DESCRIPTION
**Pull request by nedmsmith**
Addresses issue #303 consensus regarding anonymous mkey entries.

### CODE REVIEW

**Review by deeglaze — APPROVED** [review]

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -1902,17 +1903,9 @@ Conditional Endorsement Triple Transformation :
...
-
-* For each entry in the Conditional Endorsement Triple `endorsements` array, the `endorsed-triple-record` is copied to an `addition` ECT in the `ev` entry.
-
-* For each `measurement-map` entry in the measurements list of the `endorsed-triple-record`, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `addition` ECT `element-map` of the `element-list`.
+* For each entry in the Conditional Endorsement Triple ({{sec-comid-triple-cond-endors}}) `conditions` list, the `stateful-environment-record` `environment` is copied to the `environment` of the `ev` `condition` ECT and the `stateful-environment-record` `claims-list` entries are copied to the `element-list` of the `ev` `condition` ECT.
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/303
+* For each entry in the Conditional Endorsement Triple `endorsements` list, the `endorsed-triple-record` `condition` `environment-map` is copied to the `environment` of the `ev` `addition` ECT and the `endorsed-triple-record` `endorsement` list entries are copied to the `element-list` entries of the `ev` `addition` ECT.
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:1908]: A tough nut to parse.  I am wondering whether pseudo-code would be a better fit for describing this kind of stuff.

**Review by thomas-fossati — APPROVED** [review]
LGTM

I have left a non-blocking observation on readability

File: draft-ietf-rats-corim.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-ietf-rats-corim.md:1908]: for example, something like (to be checked):
```
foreach e in CET:
  copy(e.endorsed-triple-record.condition.environment-map, ECT.ev.addition.environment)
  copy(e.endorsed-triple-record.entries, ECT.ev.addition.element-list)
```

Comment (nedmsmith) [reply draft-ietf-rats-corim.md:1908]: Is dot notation universal enough that readers won't be confused by it? For example, the CDDL RFC says "CDDL allows the specification of a data item type by referring to the
   CBOR representation (specifically, to major types and additional
   information; see [Section 2 of [RFC7049]](https://www.rfc-editor.org/rfc/rfc7049#section-2))."
But I don't think that text is describing this convention.

Comment (nedmsmith) [reply draft-ietf-rats-corim.md:1908]: I might favor using back quotes for each CDDL expression so the dot isn't inside of a back quoted section. Such as:

`e`.`endorsed-triple-record`.`condition`.`environment-map`

Comment (nedmsmith) [reply draft-ietf-rats-corim.md]: Addressed in commit https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/315/commits/d43b90eff26c3abea4ae8879b0ee47a036612b2e

**Review by henkbirkholz — APPROVED** [review]
thanks for all the polish!

**Pull request merged**